blockchyp/blockchyp_client.py

```diff
@@ -3,20 +3,23 @@
 """
 
 
+from datetime import datetime
 import requests
+from .blockchyp_crypto import CryptoUtils
```

Comment:
You've already imported CryptoUtils into the global namespace, so you should be able to import it as `from blockchyp import CryptoUtils`

Reply:
tox throws an error when I do this